rebar.config

```diff
@@ -33,14 +35,14 @@
 
 {xref_checks, [
                undefined_function_calls,
-               undefined_functions,
+               undefined_functions
                %locals_not_used,
-               deprecated_function_calls,
-               deprecated_functions
+               %deprecated_function_calls,
+               %deprecated_functions
```

Comment:
assume this is temporary since there's a fair bit of code this PR is going to deprecate that we're not immediately ready to delete